src/SideNav.tsx

```diff
@@ -40,8 +47,15 @@ const SideNav = styled(SideNavBase)`
   ${COMMON};
   ${sx};
 `
-
-SideNav.Link = styled(Link).attrs(props => {
+type StyledSideNavLinkProps = {
+  to?: History.LocationDescriptor
+  selected?: boolean
+  variant?: 'full' | 'normal'
+} & SystemCommonProps &
+  SxProp &
+  SystemTypographyProps
```

Comment:
I think `common`, `typography`, and `sx` are already in the type definition for `Link`
```suggestion
}
```